### ISSUE TITLE
AltTab doesnt work with Arc in focus

### ISSUE THREAD

**Issue by Mathiasko** [lwouis/alt-tab-macos#3410]
if any other app is in focus AltTab works normally, when switched to Arc browser, hitting cmd + tab doesn't do anything

versions:
macos: 14.5
altTab: 6.70.1

reproducibility 100%

**Comment by tails618** [comment]
For what it's worth on AltTab 6.70.1, Arc 1.47.1, and macOS 15.0 (Sequoia dev beta), I *cannot* reproduce this issue (and have never seen it before, including on macOS 14.x).

**Comment by liangyi9812** [comment]
same issue
Arc: Little Arc window
[LyricsX](https://github.com/ddddxxx/LyricsX): Lyrics window

macos: 14.5
altTab: 6.70.1

Arc: 1.49.0
LyricsX : 1.6.3
![SCR-20240628-pogf-2](https://github.com/lwouis/alt-tab-macos/assets/6373858/0b25df90-1a4f-457b-b230-d7555d490031)

**Comment by tails618** [comment]
@liangyi9812 your issue seems different from the original issue. I think the original issue was that AltTab didn't work at all when Arc was in focus (so if Arc was focused, the keyboard shortcut wouldn't do anything). Your issue seems to be that the Little Arc window (and the LyricsX window) don't show in AltTab.

**Comment by papey** [comment]
> @liangyi9812 your issue seems different from the original issue. I think the original issue was that AltTab didn't work at all when Arc was in focus (so if Arc was focused, the keyboard shortcut wouldn't do anything). Your issue seems to be that the Little Arc window (and the LyricsX window) don't show in AltTab.

Hi @tails618, I also have the same issue, maybe we should open a dedicated issue for this one ?

**Comment by lwouis** [comment]
https://github.com/lwouis/alt-tab-macos/issues/4192

**Comment by brianjbishop** [comment]
i'm having a similar issue on Mac, can't use Option Tab as a shortcut, kind of frustrating

**Comment by shp7724** [comment]
I don’t think this issue exists only on the Arc side. When I restart AltTab, apps that didn’t appear before start showing up.
